cookiecutter-django-ida/{{cookiecutter.repo_name}}/docs/conf.py

```diff
@@ -0,0 +1,512 @@
+import re
+import sys
+from subprocess import check_call
+
+import edx_theme
+from django import setup as django_setup
+
+
+def get_version(*file_paths):
+    """
+    Extract the version string from the file.
+
+    Input:
+     - file_paths: relative path fragments to file with
+                   version string
+    """
+    filename = os.path.join(os.path.dirname(__file__), *file_paths)
+    version_file = open(filename, encoding="utf8").read()
+    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]", version_file, re.M)
+    if version_match:
+        return version_match.group(1)
+    raise RuntimeError('Unable to find version string.')
+
+
+REPO_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(REPO_ROOT)
+
+VERSION = get_version('../{{ cookiecutter.project_name }}', '__init__.py')
+
+# Configure Django for autodoc usage
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', '{{ cookiecutter.project_name }}.settings.test')
+django_setup()
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'edx_theme',
+    'sphinx.ext.autodoc',
+    'sphinx.ext.doctest',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.ifconfig',
+    'sphinx.ext.napoleon'
+]
+
+# A list of warning types to suppress arbitrary warning messages.
+suppress_warnings = [
+    'image.nonlocal_uri',
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The encoding of source files.
+#
+# source_encoding = 'utf-8-sig'
+
+# The top level toctree document.
+top_level_doc = 'index'
+
+# General information about the project.
+project = '{{ cookiecutter.project_name }}'
+copyright = edx_theme.COPYRIGHT  # pylint: disable=redefined-builtin
+author = edx_theme.AUTHOR
+project_title = '{{ cookiecutter.project_name }}'
+documentation_title = f"{project_title}"
+
+
+# Set display_github to False if you don't want "edit on Github" button
+html_context = {
+    "display_github": True,  # Integrate GitHub
+    "github_user": "edx",  # Username
+    "github_repo": '{{ cookiecutter.project_name }}',  # Repo name
+    "github_version": "main",  # Version
+    "conf_py_path": "/docs/",  # Path in the checkout to the docs root
+}
```

Comment:
This is not in the app conf.py. Did this come from elsewhere? Should it be in the app file?